### PR TITLE
Magic number cleanup part 6

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**

### CODE REVIEW

File: packages/app/src/components/time-series-chart/components/timeline/components/tooltip-content.tsx [hunk 1]
```diff
@@ -16,27 +17,16 @@ interface TimelineTooltipContentProps {
...
-    .join(' – ');
+  const dateStr = [formatDateFromSeconds(config.start, 'medium'), config.end && formatDateFromSeconds(config.end, 'medium')].filter(isDefined).join(' – ');
 
   return (
     <Box
       color="black"
-      px={18}
-      py={15}
+      px="18p"
+      py="15px"
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/components/time-series-chart/components/timeline/components/tooltip-content.tsx:29]: Shouldn't this be as below?
```
paddingX="18px"
paddingY="15px"
```

Or, even shorthanded as `padding="15px 18px"`?

File: packages/app/src/components/time-series-chart/components/timeline/components/tooltip-content.tsx [hunk 2]
```diff
@@ -85,15 +58,7 @@ export function TimelineTooltipContent({
...
-        <Box
-          pt={3}
-          mx={-27}
-          borderTop="1px solid"
-          borderTopColor="gray2"
-          display="flex"
-          justifyContent="center"
-          textVariant="label1"
-        >
+        <Box paddingTop={space[3]} marginX={'-27px'} borderTop="1px solid" borderTopColor="gray2" display="flex" justifyContent="center" textVariant="label1">
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/components/time-series-chart/components/timeline/components/tooltip-content.tsx:61]: See my earlier remarks regarding combining static values with `{ ... }`.

File: packages/app/src/components/time-series-chart/time-series-chart.tsx [hunk 3]
```diff
@@ -187,7 +188,7 @@ export function TimeSeriesChart<T extends TimestampedValue, C extends SeriesConf
     width,
     height,
     paddingLeft,
-    paddingTop: showWeekNumbers ? 20 : undefined,
+    paddingTop: showWeekNumbers ? 20 : undefined, // Still a magic number, but this is not easaly possible without refactoring the useDimensions function
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/components/time-series-chart/time-series-chart.tsx:191]: 🙌

File: packages/app/src/components/time-series-chart/time-series-chart.tsx [hunk 4]
```diff
@@ -395,7 +396,7 @@ export function TimeSeriesChart<T extends TimestampedValue, C extends SeriesConf
              * Highlight 0 on the y-axis when there are positive and
              * negative values
              */}
-            {highlightZero && <rect x={0} y={yScale(0)} width={bounds.width} height={1} fill="black" />}
+            {highlightZero && <rect x={0} y={yScale(0)} width={bounds.width} height={'1px'} fill="black" />}
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/components/time-series-chart/time-series-chart.tsx:399]: See my earlier remarks regarding combining static values with `{ ... }`.

File: packages/app/src/components/time-series-chart/components/bar-trend.tsx [hunk 5]
```diff
@@ -81,23 +64,8 @@ export function BarTrend({
...
-                <rect
-                  key={index}
-                  x={x}
-                  y={y}
-                  height={barHeight}
-                  width={barWidth}
-                  fill={'url(#diagonal-pattern)'}
-                  id={id}
-                />
+                <PatternLines id="diagonal-pattern" height={6} width={6} stroke={colors.neutral} strokeWidth={2} orientation={['diagonal']} />
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/components/time-series-chart/components/bar-trend.tsx:67]: `height={6} width={6}` should be in pixel values

File: packages/app/src/components/time-series-chart/components/timespan-annotation.tsx [hunk 6]
```diff
@@ -51,36 +46,13 @@ export function TimespanAnnotation({
...
-          patternUnits="userSpaceOnUse"
-        >
-          <line
-            x1="0"
-            y1="0"
-            x2="0"
-            y2="8"
-            style={{ stroke: 'white', strokeWidth: 4 }}
-          />
+        <pattern id={patternId} width="8" height="8" patternTransform="rotate(-45 0 0)" patternUnits="userSpaceOnUse">
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/components/time-series-chart/components/timespan-annotation.tsx:49]: `width="8" height="8"` should be pixel values?

File: packages/app/src/components/time-series-chart/components/timespan-annotation.tsx [hunk 7]
```diff
@@ -51,36 +46,13 @@ export function TimespanAnnotation({
...
-          patternUnits="userSpaceOnUse"
-        >
-          <line
-            x1="0"
-            y1="4"
-            x2="0"
-            y2="0"
-            style={{ stroke: 'white', strokeWidth: 4, strokeDasharray: 2 }}
-          />
+        <pattern id={patternId} width="4" height="4" patternUnits="userSpaceOnUse">
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/components/time-series-chart/components/timespan-annotation.tsx:54]: `width="4" height="4"` should be pixel values?

**Review by VWSCoronaDashboard28 — CHANGES_REQUESTED** [review]

File: packages/app/src/components/time-series-chart/time-series-chart.tsx (hunk 3 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/time-series-chart/time-series-chart.tsx:191]: Thanks for noticing and valuing my comment!

File: packages/app/src/components/time-series-chart/components/timeline/components/tooltip-content.tsx (hunk 1 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/time-series-chart/components/timeline/components/tooltip-content.tsx]: adjusted accordingly, please resolve.

File: packages/app/src/components/time-series-chart/components/timeline/components/tooltip-content.tsx (hunk 2 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/time-series-chart/components/timeline/components/tooltip-content.tsx]: adjusted accordingly, please resolve.

File: packages/app/src/components/time-series-chart/time-series-chart.tsx (hunk 4 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/time-series-chart/time-series-chart.tsx:399]: adjusted accordingly, please resolve.

File: packages/app/src/components/time-series-chart/components/bar-trend.tsx (hunk 5 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/time-series-chart/components/bar-trend.tsx:69]: This is a VisX number type requirement

File: packages/app/src/components/time-series-chart/components/timespan-annotation.tsx (hunk 7 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/time-series-chart/components/timespan-annotation.tsx]: See previous comments

File: packages/app/src/components/time-series-chart/components/timespan-annotation.tsx (hunk 6 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/time-series-chart/components/timespan-annotation.tsx]: see previous comments

**Review by VWSCoronaDashboard26 — APPROVED** [review]
Pre-approving as all my earlier remarks have been resolved.

**Pull request merged**